app/views/components/_notice.html.erb

```diff
@@ -3,8 +3,15 @@
     description_text ||= ""
     description_govspeak ||= ""
   %>
+
   <section class="app-c-notice" aria-label="Notice">
-    <h2 class="app-c-notice__title"><%= title %></h2>
+
+    <% if description_text.present? || description_govspeak.present? %>
+      <h2 class="app-c-notice__title"><%= title %></h2>
+    <% else %>
+      <p class="app-c-notice__title"><%= title %></p>
+    <% end %>
+
     <% if description_text.present? %>
       <p class="app-c-notice__description"><%= description_text %></p>
     <% end %>
```

Comment:
Sorry, I was confused by the way github displays pages. Looking at the complete file it looked like there was an extra tab on the left of every line, but there isn't, so never mind.